sharvahl/Weapons_Master_Rahoul.pl

```diff
@@ -40,6 +40,9 @@ sub EVENT_ITEM {
   }
   if (plugin::check_handin(\%itemcount, 9053 => 2, 9054 => 1, 9052 => 1)) {
     quest::say("Ah, I see you have been so kind as to help us. Muada would like you take your assistance one step further. If you are [" . quest::saylink("willing") . "], of course.");
+    if($client->GetGlobal("beast_turnin") ==undef) {
```

Comment:
```suggestion
    if($client->GetGlobal("beast_turnin") ==undef) {
```
Since the global check is inside this block, the player has now lost the items if this check fails. Recommend moving in front of the plugin check or add an else statement to return the 3 items.